src/Common/src/TypeSystem/Ecma/EcmaType.cs

```diff
@@ -537,6 +537,85 @@ private MarshalAsDescriptor GetMarshalAsDescriptor(FieldDefinition fieldDefiniti
             return null;
         }
 
+        public override PInvokeAttributes GetDelegatePInvokeAttributes()
+        {
+            PInvokeAttributes attributes = PInvokeAttributes.None;
+            if (!this.IsDelegate)
+            {
+                return attributes;
+            }
+
+            var customAttributeValue = this.GetDecodedCustomAttribute(
+                               "System.Runtime.InteropServices", "UnmanagedFunctionPointerAttribute");
+
+            if (customAttributeValue == null)
+            {
+                return attributes;
```

Comment:
This would be a good time to add the default value in. You'd want to keep the understanding of default values in one place.